omxplayer/player.py

```diff
@@ -574,6 +574,17 @@ def set_position(self, position):
         self._player_interface.SetPosition(ObjectPath("/not/used"), Int64(position * 1000.0 * 1000))
         self.positionEvent(self, position)
 
+    @_check_player_is_active
+    @_from_dbus_type
+    def set_layer(self, layer):
+        """
+        Set the layer of the Video
```

Comment:
Might be worth saying what the default layer is, and whether higher layers are above lower layers or vice versa